bogie/process.go

```diff
@@ -90,10 +91,23 @@ func genContext(envfile string) (*context, error) {
 	return &c, nil
 }
 
+func fileExists(filename string) bool {
+	info, err := os.Stat(filename)
+	if os.IsNotExist(err) {
+		return false
+	}
+	return !info.IsDir()
+}
 func setValueContext(app *ApplicationInput, old *context) (*context, error) {
 	c := context{}
 
 	files := []string{}
+	if val, ok := old.Values["default_region"]; ok {
```

Comment:
Prefer a variable name like `default_region` over `val` as it is more specific to what it contains.

Reply:
made the variable name defaultRegion